app/views/dashboard/_eeo_data.haml

```diff
@@ -4,7 +4,7 @@
     .note_item
       = select_tag :department_ids,
       options_from_collection_for_select(@departments, :id, :name),
-      class: 'dept_select', multiple: true
+      class: 'dept-select', multiple: true
```

Comment:
As I said before, I'd like for it to have an underscore since most of our other CSS class names have underscores. You should only need to make it match everywhere.